Earthfile

```diff
@@ -0,0 +1,6 @@
+lint:
```

Comment:
It's possible to do the matrix testing within earthly itself. For example you could define a target like this:

```
lint-all:
  BUILD +lint --EMACS_VERSION=26 --EMACS_VERSION=27
```